university-de-dschang.csl

```diff
@@ -0,0 +1,719 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" page-range-format="expanded">
```

Comment:
Which languages will this style be used for?

Reply:
French and English

Reply:
changed "es" to "fr"